tests/map/options.test.ts

```diff
@@ -30,4 +31,30 @@ describe('MapOptions', () => {
         assert(document.body.innerHTML === '');
         assert(root.innerHTML === '');
     });
+
+    it('scale label for min/max switches between linear and log', () => {
+        const root = document.createElement('div');
+        const options = new MapOptions(root, DUMMY_PROPERTIES, DUMMY_CALLBACK);
+
+        assertScaleLabel(options.x, 'x');
+        assertScaleLabel(options.y, 'y');
+        assertScaleLabel(options.z, 'z');
+
+        function assertScaleLabel(axisOptions: AxisOptions, axisName: string): void {
+            const min = getByID(`chsp-${axisName}-min-label`);
+            const max = getByID(`chsp-${axisName}-max-label`);
+
+            // change from linear (default) to log scale
+            axisOptions.scale.value = 'log';
```

Comment:
instead of manually settings the value & calling the function, could you simulate user interaction here by setting the scale through the HTML select element?